### ISSUE TITLE
Jerking waveform

### ISSUE THREAD

**Issue by tbazant** [mixxxdj/mixxx#13141]
### Bug Description

Moving mouse cursor from library to waveform makes the running waveform jerk. It happens when the cursor is at the lower edge of the waveform pane (the one that enables you to stretch waveform's height). Happens on both X and Wayland, NVidia native or Nouveau drivers

### Version

2.4.0

### OS

Debian 12, Debian testing (flathub version)

**Comment by JoergAtGithub** [comment]
You could try if the experimental VSync mode PLL described here https://github.com/mixxxdj/mixxx/wiki/Changing-the-VSync-Mode-for-Scrolling-Waveforms improves this.

**Comment by tbazant** [comment]
> You could try if the experimental VSync mode PLL described here https://github.com/mixxxdj/mixxx/wiki/Changing-the-VSync-Mode-for-Scrolling-Waveforms improves this.

just tried all possible values, the result is the same. To be precise, the leap occurs only when the mouse hovers the edge that allows resizing the waveform (or when i'm actually resizing it), otherwise everything is just fine and i can live with it.

**Comment by JoergAtGithub** [comment]
I see this on Windows 11 too

**Comment by JoergAtGithub** [comment]
You could try the build from PR #12904. The Link _Ubuntu 22.04 DEB_ at the bottom of https://github.com/mixxxdj/mixxx/actions/runs/8123807324?pr=12904
It prevents some kind of waveform jerking, but I guess this here is an additional problem.

**Comment by m0dB** [comment]
It is expected that resizing causes the animation to be less smooth. It is surprising that simply hovering over the resize  handle also causes this. On macOS I can't reproduce this...

**Comment by m0dB** [comment]
When you quickly move the mouse over it does this also happen, or do you have wait a bit when hovering? Is it all the time while hovered over the splitter or just a single hiccup?

**Comment by tbazant** [comment]
i reverted back to 2.3.6 DEB version and there it does not happen.

**Comment by tbazant** [comment]
> When you quickly move the mouse over it does this also happen, or do you have wait a bit when hovering? Is it all the time while hovered over the splitter or just a single hiccup?

It's only one-time hiccup and yes, even when moving quickly

**Comment by JoergAtGithub** [comment]
I guess one reason are the ToolTips. @tbazant Could you try to disable the tooltips in the preferences. Does this improve the waveforms?

**Comment by tbazant** [comment]
I've been using Mixxx 2.4.1 / Flatpak version for a while and the problem is gone, no waveform jerking occurs when i move mouse anywhere across the app GUI. With enabled tooltips.

**Comment by mxmilkiib** [comment]
would this be resolved?

**Comment by acolombier** [comment]
Likely still a problem - @tbazant do you still experience the same issue on 2.5 or the 2.6 beta?

**Comment by tbazant** [comment]
No, i have been using Linux self-compiled version of 2.6 branch and the problem is gone. Thanks!
